.eslintrc.yml

```
@@ -1,4 +1,4 @@
 plugins: [jsdoc]
-extends: [cheminfo, 'plugin:jsdoc/recommended']
+extends: [cheminfo-typescript, 'plugin:jsdoc/recommended']
```

```suggestion
extends: [cheminfo-typescript, cheminfo-typescript/jsdoc]
```

Our custom config is better for TypeScript support (types should not be duplicated in the jsdoc comment)

---

fixed